app/views/dashboard/claims/index.html.erb

```diff
@@ -0,0 +1,66 @@
+<section class="section" id="content">
+    <div class="container">
+        <h2><strong>Approved:</strong></h2><br>
+        <div class="column has-text-centered">
+        <% if @claims.empty? %>
+            <p> <%= t('.no_giveaways') %> </p>
+        <%else %>
+            <div class="columns is-mobile is-multiline is-left">
+                <% @claims.each do |giveaway|%>
```

Comment:
if you do `@claims` each, then the local variable should be names `claim` not `giveaway`.